website/Portfolio.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Omar Alhazmi | Portfolio </title>
+
+</head>
+<body>
+        <header>
+            <ul class="nul">
+                <li class="nli"><a  href="/Users/mac/sei/homeworks/hw-w02d01-website/website/AboutMe.html">About Me</a></li>
+                <li class="nli"><a class="active"  href="/Users/mac/sei/homeworks/hw-w02d01-website/website/Portfolio.html">Portfolio</a></li>
+              
+              </ul>        
+            </header>
+        <main>
+                <h2> Omar Alhazmi</h2>    
+         </main>
+    <h3>Experience</h3>
```

Comment:
be more consistant with you indentation to make your code more readable

here is a link for more information

https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html